### PR TITLE
#158747142 API endpoint Create user accounts that can signin/signout 

### PR DESCRIPTION
**Pull request by rethes**
#### What does this PR do?
Create endpoints for sign up and log in

#### Description of Task to be completed?
Add validators for the user details
Register users routes with Blueprints
Add sign in endpoints
Add log in endpoints
Add a database for the tests
Update tests

#### How should this be manually tested?
Using Postman

#### Any background context you want to provide?
Postgres Database in use

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/158747142
https://www.pivotaltracker.com/story/show/158531519

### CODE REVIEW

File: myapp/test_database.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    # calling the database connection
+    conn = db()
+    # Open a cursor to perform database operations
+    with conn.cursor() as cursor:
+        # Execute a command: this creates a new table
+        cursor.execute(query1)
+        # Make the changes to the database persistent
+        conn.commit()
+
+    query2 = "DROP TABLE IF EXISTS rides;" \
```
Comment (JoshuaOndieki) [inline myapp/test_database.py]: Maybe giving your queries some meaningful names would be nice

**Comment by JoshuaOndieki** [comment]
Your tests are failing, does this mean this is still a work in progress?

File: myapp/test_database.py (hunk 1 above)
Comment (rethes) [reply myapp/test_database.py]: Noted.

Comment (rethes) [reply myapp/test_database.py]: Yes, this is still a work in progress

**Pull request merged**